examples/example7.c++

```diff
@@ -0,0 +1,79 @@
+#define NONIUS_RUNNER
+#include <nonius.h++>
+
+#include <string>
+#include <vector>
+#include <cmath>
+
+NONIUS_PARAM("x", 1.0)
+
+template <typename Fn>
+struct volatilize_fn {
```

Comment:
I suppose this is here because of #52, right? If it works, I'd say it should be part of what nonius does when running the code.
I have been researching a bit on optimizer control, and I put up some code for that (#57). If you could have a look at it, that'd be great.